nbs/15_gsa_gls/05-snps_into_chr_cov.ipynb

```diff
@@ -215,69 +381,13 @@
     },
     "tags": []
    },
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Processing mashr_Skin_Not_Sun_Exposed_Suprapubic.db\n",
-      "Processing mashr_Cells_EBV-transformed_lymphocytes.db\n",
-      "Processing mashr_Brain_Frontal_Cortex_BA9.db\n",
-      "Processing mashr_Kidney_Cortex.db\n",
-      "Processing mashr_Brain_Substantia_nigra.db\n",
-      "Processing mashr_Spleen.db\n",
-      "Processing mashr_Colon_Transverse.db\n",
-      "Processing mashr_Heart_Left_Ventricle.db\n",
-      "Processing mashr_Lung.db\n",
-      "Processing mashr_Muscle_Skeletal.db\n",
-      "Processing mashr_Brain_Hypothalamus.db\n",
-      "Processing mashr_Brain_Cortex.db\n",
-      "Processing mashr_Brain_Amygdala.db\n",
-      "Processing mashr_Esophagus_Mucosa.db\n",
-      "Processing mashr_Adrenal_Gland.db\n",
-      "Processing mashr_Uterus.db\n",
-      "Processing mashr_Prostate.db\n",
-      "Processing mashr_Whole_Blood.db\n",
-      "Processing mashr_Pituitary.db\n",
-      "Processing mashr_Esophagus_Gastroesophageal_Junction.db\n",
-      "Processing mashr_Stomach.db\n",
-      "Processing mashr_Heart_Atrial_Appendage.db\n",
-      "Processing mashr_Brain_Cerebellum.db\n",
-      "Processing mashr_Breast_Mammary_Tissue.db\n",
-      "Processing mashr_Artery_Tibial.db\n",
-      "Processing mashr_Artery_Aorta.db\n",
-      "Processing mashr_Small_Intestine_Terminal_Ileum.db\n",
-      "Processing mashr_Brain_Hippocampus.db\n",
-      "Processing mashr_Testis.db\n",
-      "Processing mashr_Brain_Putamen_basal_ganglia.db\n",
-      "Processing mashr_Pancreas.db\n",
-      "Processing mashr_Adipose_Subcutaneous.db\n",
-      "Processing mashr_Brain_Cerebellar_Hemisphere.db\n",
-      "Processing mashr_Colon_Sigmoid.db\n",
-      "Processing mashr_Minor_Salivary_Gland.db\n",
-      "Processing mashr_Cells_Cultured_fibroblasts.db\n",
-      "Processing mashr_Brain_Nucleus_accumbens_basal_ganglia.db\n",
-      "Processing mashr_Brain_Anterior_cingulate_cortex_BA24.db\n",
-      "Processing mashr_Vagina.db\n",
-      "Processing mashr_Ovary.db\n",
-      "Processing mashr_Skin_Sun_Exposed_Lower_leg.db\n",
-      "Processing mashr_Esophagus_Muscularis.db\n",
-      "Processing mashr_Brain_Spinal_cord_cervical_c-1.db\n",
-      "Processing mashr_Artery_Coronary.db\n",
-      "Processing mashr_Thyroid.db\n",
-      "Processing mashr_Brain_Caudate_basal_ganglia.db\n",
-      "Processing mashr_Adipose_Visceral_Omentum.db\n",
-      "Processing mashr_Nerve_Tibial.db\n",
-      "Processing mashr_Liver.db\n"
-     ]
-    }
-   ],
+   "outputs": [],
```

Comment:
I assume this will be something like   "Processing mashr_Skin_Not_Sun_Exposed_Suprapubic.db\n",
      "Processing mashr_Cells_EBV-transformed_lymphocytes.db\n",
      "Processing mashr_Brain_Frontal_Cortex_BA9.db\n"," etc. when the notebook is run on actual data again?

Reply:
We already talked about this, but for the records: the outputs are in folder `gene_corrs`, where the notebooks that you reviewed are run on different cohorts. I'm sorry if this was not clear in the description!